core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/config/UserStoreConfigXMLProcessor.java

```diff
@@ -174,6 +175,10 @@ public RealmConfiguration buildUserStoreConfiguration(OMElement userStoreElement
 //            throw new UserStoreException("Invalid domain name provided");
 //        }
 
+        if (UserStoreManagerRegistry.getUserStoreProperties(userStoreClass) == null) {
+            log.error(userStoreClass + " not found in the user store manager registry");
+            return null;
```

Comment:
shall we throw an exception here instead of returning null.

Reply:
User store properties of the User store manager registry excepted to be null  during the migration. Therefore, should not throw an exception due to that reason.